File: haina/codes/beluga/server/beluga/src/com/haina/beluga/dto/WeatherDto.java
package com.haina.beluga.dto;

import com.haina.beluga.core.dto.IDto;
import com.haina.beluga.core.util.BeanUtil;
import com.haina.beluga.domain.Weather;

public class WeatherDto implements IDto{

	/**
	 * 
	 */
	private static final long serialVersionUID = -4673967747869811297L;
	
	private String date;/*天气日期*/ 
	private String weatherCityCode; /*城市代码*/
	private String weatherType; /*天气类型*/
	private String wind; /*风速*/
	private String temperature;/*实时温度*/
	private String icon;/*天气图片URI*/
	private boolean isNight;/*是否夜里*/
	private int high; /*最高气温*/
	private int low;/*最低气温*/
	private String issuetime;/*发布时间*/
	
	public String getDate() {
		return date;
	}
	public void setDate(String date) {
		this.date = date;
	}
	public String getWeatherCityCode() {
		return weatherCityCode;
	}
	public void setWeatherCityCode(String weatherCityCode) {
		this.weatherCityCode = weatherCityCode;
	}
	public String getWeatherType() {
		return weatherType;
	}
	public void setWeatherType(String weatherType) {
		this.weatherType = weatherType;
	}
	public int getHigh() {
		return high;
	}
	public void setHigh(int high) {
		this.high = high;
	}
	public int getLow() {
		return low;
	}
	public void setLow(int low) {
		this.low = low;
	} 
	
	
	public String getWind() {
		return wind;
	}
	public void setWind(String wind) {
		this.wind = wind;
	}
	public String getTemperature() {
		return temperature;
	}
	public void setTemperature(String temperature) {
		this.temperature = temperature;
	}
	
	public String getIcon() {
		return icon;
	}
	public void setIcon(String icon) {
		this.icon = icon;
	}
	
	public boolean isNight() {
		return isNight;
	}
	public void setNight(boolean isNight) {
		this.isNight = isNight;
	}
	
	public String getIssuetime() {
		return issuetime;
	}
	public void setIssuetime(String issuetime) {
		this.issuetime = issuetime;
	}
	public static WeatherDto valueof(Weather w){
		WeatherDto dto = new WeatherDto();
		BeanUtil.copyPropertie(w, dto);
		return dto;
	}
	

}

File: trunk/haina/codes/beluga/client/moblie/beluga_net/beluga_mobile_x/src/dto/WeatherDto.cpp
#include "WeatherDto.h"


void WeatherDto::setDate(string aDate)
{
	date = aDate;
}
string WeatherDto::getDate()
{
	return date;
}

void WeatherDto::setWeatherCityCode(string aCityCode)
{
	weatherCityCode = aCityCode;
}
string WeatherDto::getWeatherCityCode()
{
	return weatherCityCode;
}

void WeatherDto::setWeatherType(string aWeatherType)
{
	weatherType = aWeatherType;
}
string WeatherDto::getWeatherType()
{
	return weatherType;
}

void WeatherDto::setWind(string aWind)
{
	wind = aWind;
}
string WeatherDto::getWind()
{
	return wind;
}

void WeatherDto::setTemperature(string aTemperature)
{
	temperature = aTemperature;
}
string WeatherDto::getTemperature()
{
	return temperature;
}

void WeatherDto::setIcon(string aIconPath)
{
	icon = aIconPath;
}
string WeatherDto::getIcon()
{
	return icon;
}

/*
void WeatherDto::setIsNight(bool aIsNight)
{
	isNight = aIsNight;
}
bool WeatherDto::getIsNight()
{
	return isNight;
}*/


void WeatherDto::setHigh(int aHigh)
{
	high = aHigh;
}
int	 WeatherDto::getHigh()
{
	return high;
}

void WeatherDto::setLow(int aLow)
{
	low = aLow;
}
int	 WeatherDto::getLow()
{
	return low;
}

void WeatherDto::setIssuetime(string aIssuetime)
{
	issuetime = aIssuetime;
}
string WeatherDto::getIssuetime()
{
	return issuetime;
}
